test/app.test.js

```diff
@@ -0,0 +1,8 @@
+const { validateArgvExist, validateExt, readFiles, selectOption, getLinks, validateLinks, stats } = require('../app')
```

Comment:
veo la invocación pero no la implementación 
```suggestion
const { validateArgvExist, validateExt, readFiles, selectOption, getLinks, validateLinks, stats } = require('../app')
```